app/config/config_dev.yml

```diff
@@ -14,15 +14,18 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
-            path:  %kernel.logs_dir%/%kernel.environment%.log
-            level: debug
+            type:   stream
+            path:   %kernel.logs_dir%/%kernel.environment%.log
+            level:  debug
+        console:
+            type:   console
+            bubble: false
```

Comment:
I already changed it in my app ; But I wanted to know if you think it's a good idea to change it in symfony too.